e2e/_suites/metricbeat/metricbeat_test.go

```diff
@@ -408,7 +410,7 @@ func (mts *MetricbeatTestSuite) runMetricbeatService() error {
 
 		err = docker.TagImage(
 			"docker.elastic.co/beats/metricbeat:"+metricbeatVersionBase,
-			"docker.elastic.co/observability-ci/metricbeat:"+mts.Version+"-amd64",
+			"docker.elastic.co/observability-ci/metricbeat:"+mts.Version+"-"+arch,
```

Comment:
Docker images are pushed to the docker registry, and their tag name contains the architecture, for further details refers to elastic/beats#24121